### PR TITLE
vim-patch:3c053a1a5ad2

### PR DESCRIPTION
**Pull request by clason**
Update runtime files
https://github.com/vim/vim/commit/3c053a1a5ad2a3c924929e11f2b9af20a8b901e2

### CODE REVIEW

**Review by zeertzjq — COMMENTED** [review]
Missing:
- `filetype.vim` date update
- `optwin.vim` (can be ported without `gettext()`)

Not sure about package-related docs.

**Comment by zeertzjq** [comment]
`test/functional/plugin/health_spec.lua` needs a small change.

**Comment by clason** [comment]
> * `filetype.vim` date update

That file will be removed anyway; I thought I'd save myself one rebase. But sure.

> Not sure about package-related docs.

Not sure either. Bfredlpackages are almost-but-not-entirely-like Brampackages.

**Comment by seandewar** [comment]
Package stuff seems to still work for the things mentioned (probably due to `do_in_path_and_pp`).

**Comment by clason** [comment]
Yeah, but does it work exactly the same way? (Meaning, as separate from `rtp`?)

**Comment by seandewar** [comment]
Seems so. I tested by clearing the `rtp` before the init file was loaded and it was still able to source color schemes from start/opt packages, and autoload functions from start packages only (unless they were also removed from the `packpath`).

File: runtime/doc/userfunc.txt [hunk 1]
```diff
@@ -388,6 +388,11 @@ file should then define the function like this: >
 	   echo "Done!"
 	endfunction
 
+
```
Comment (seandewar) [inline runtime/doc/userfunc.txt:391]:
```suggestion
```

**Review by seandewar — APPROVED** [review]

**Review by zeertzjq — APPROVED** [review]

**Pull request merged**